.travis.yml

```diff
@@ -11,25 +11,30 @@ script:
 
 jobs:
   include:
-  - stage: Unit Test + Code Style Fixing
-    php: 5.3
-  - stage: Unit Test + Code Style Fixing
-    php: 5.4
-  - stage: Unit Test + Code Style Fixing
-    php: 5.5
-  - stage: Unit Test + Code Style Fixing
-    php: 7.0
-  - stage: Unit Test + Code Style Fixing
+  - stage: Unit Tests + Code Style Fixing + Code Coverage
     php: 7.1
-  - stage: Unit Test + Code Style Fixing
-    php: 5.6
+    before_script:
+    - composer require --dev php-coveralls/php-coveralls
+    - composer install
     script:
+    - vendor/bin/phpunit --coverage-clover build/logs/clover.xml
     - curl -L http://files.shopgate.com/travis-ci/php-cs-fixer.phar -o php-cs-fixer
     - chmod a+x php-cs-fixer
     - git checkout $TRAVIS_BRANCH
     - ./php-cs-fixer fix --config=.php-cs.dist --cache-file=.php-cs.cache --diff --dry-run --verbose .
     - rm php-cs-fixer
-    - vendor/bin/phpunit
+    after_success:
+    - travis_retry php vendor/bin/php-coveralls -v
+  - stage: Unit Tests
+    php: 5.3
+  - stage: Unit Tests
+    php: 5.4
+  - stage: Unit Tests
+    php: 5.5
+  - stage: Unit Tests
+    php: 5.6
```

Comment:
we should confirm with PO, but I'd dump anything below 5.6

Reply:
@Stereck approved to dump everything below 5.6.

Reply:
But we should also add php 7.2 imho